### DISCUSSION TITLE
Daily 14/06

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#617] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Ayer hice varios arreglos, boton para carrito y en la clase nos dijo que el back-end para pagar debería ser más complicado de lo que pensé así que creo que voy a hacer algo rápido en node.

**Comment by BraianTroncoso** [comment]
Hoy voy a realizar la issue de ML, si puedo terminarla voy a asignarme mas tareas.

**Comment by NataliaAlvarezIspc** [comment]
Ayer estuve termiando  la tarea de vaciar el carrito. El carrito se vacía pero no devuelve las cantidades seleccionadas a catálogo.
Hoy estuve estudiando inglés ya que a la noche tengo clase. No tuve mucho tiempo, así que solo pude hacer eso.
